### ISSUE TITLE
Obsolete javax.xml.bind:jaxb-api dependency

### ISSUE THREAD

**Issue by filiphr** [liquibase/liquibase#1866]
I don't know from where but `liquibase-core` contains `javax.xml.bind:jaxb-api` as a direct compile dependency. Looking in the [liquibase-core/pom.xml](https://github.com/liquibase/liquibase/blob/master/liquibase-core/pom.xml) I cannot see the dependency being set. On top of that Liquibase core uses the java `SAXParser`.

I have tried the latest release (4.3.5) in [this](https://github.com/filiphr/spring-boot-2.2-liquibase-problem/tree/liquibase-4.3) repository. That repository was an example repository that showcased the problem that existed in 3.8.x Liquibase when run under Java 11 without the jaxb-api. However, it seems like the problem has been resolved and Liquibase (at least core) runs correctly on Java 11 without the need of the jaxb-api. 

Is it possible for you to remove this dependency from liquibase-core?

**Comment by filiphr** [comment]
I did some more analysis on this and it seems like the dependency is needed by Liquibase Pro. More precisely by `DaticalTrueLicenseService`. If the dependency is exclude then there won't be a `LicenseService` and there might be a `NullPointerException` at some point.

e.g.

```
Caused by: liquibase.exception.DatabaseException: liquibase.exception.LiquibaseException: java.lang.NullPointerException
	at liquibase.Liquibase.dropAll(Liquibase.java:1804)
	at liquibase.Liquibase.dropAll(Liquibase.java:1767)
	at org.flowable.common.engine.impl.db.LiquibaseBasedSchemaManager.schemaDrop(LiquibaseBasedSchemaManager.java:116)
	... 40 more
Caused by: liquibase.exception.LiquibaseException: java.lang.NullPointerException
	at liquibase.Liquibase.runInScope(Liquibase.java:2374)
	at liquibase.Liquibase.dropAll(Liquibase.java:1783)
	... 42 more
Caused by: java.lang.NullPointerException
	at com.datical.liquibase.ext.command.RollbackOneChangeSetCommand.getPriority(Unknown Source)
	at liquibase.command.CommandFactory$1.compare(CommandFactory.java:49)
	at liquibase.command.CommandFactory$1.compare(CommandFactory.java:46)
	at java.base/java.util.TimSort.binarySort(TimSort.java:296)
	at java.base/java.util.TimSort.sort(TimSort.java:221)
	at java.base/java.util.Arrays.sort(Arrays.java:1515)
	at java.base/java.util.ArrayList.sort(ArrayList.java:1750)
	at java.base/java.util.Collections.sort(Collections.java:179)
	at liquibase.command.CommandFactory.getCommand(CommandFactory.java:55)
	at liquibase.Liquibase$15.run(Liquibase.java:1787)
	at liquibase.Scope.lambda$child$0(Scope.java:160)
	at liquibase.Scope.child(Scope.java:169)
	at liquibase.Scope.child(Scope.java:159)
	at liquibase.Scope.child(Scope.java:138)
	at liquibase.Liquibase.runInScope(Liquibase.java:2369)
	... 43 more
```

**Comment by nvoxland** [comment]
We have been working on our 3rd party dependency setup a bit, and with the latest code in master I'm finding that everything works without the jaxb dependencies anymore. Even the pro code.

I created #1882 for additional testing to make sure I didn't miss anything.

**Comment by filiphr** [comment]
That is great to hear @nvoxland.

>I thought maybe I'd need to shade them or something, but when I created #1882 for additional testing everything worked.

It would be good if we can avoid shading as the liquibase-core jar is big enough as it is already (3.2 MB currently).

**Comment by nvoxland** [comment]
With further testing, we are still needing the jaxb dependencies for the pro license management. I did find a problem where the missing jaxb libraries produce an unhelpful null pointer exception that will be fixed in #1931, but beyond that I'll have to close this as "won't do" for now.

We have plans to re-evaluate our license management at some point and reducing dependencies are part of those requirements. But that is a larger/later project.
